scripts/feature_toggle_report_generator.py

```diff
@@ -13,31 +14,41 @@
 
 @click.command()
 @click.argument(
-    'sql_dump_path',
-    type=click.Path(exists=True),
-)
-@click.argument(
-    'annotation_report_path',
+    'data_path',
     type=click.Path(exists=True),
 )
 @click.argument(
     'output_path', default="feature_toggle_report",
 )
-@click.argument(
-    'environment_name', required=True
-)
 @click.option(
-    '--show_state', is_flag=True,
+    '--show-state', is_flag=True,
 )
-def main(sql_dump_path, annotation_report_path, output_path, environment_name, show_state):
-    ida_names = ['lms']
-    idas = {name: IDA(name) for name in ida_names}
-    if show_state:
-        add_toggle_state_to_idas(idas, sql_dump_path)
-    add_toggle_annotations_to_idas(idas, annotation_report_path)
+def main(data_path, output_path, show_state):
+
+    # annotations should be located in annotations dir with file named: <ida_name>-annotations.yml
+    annotation_dir = os.path.join(data_path, "annotations")
+
+    # each env should have a folder with all its sql dump with toggle data
+    # folders name as: <env_name>_env  #TODO(jinder): should this be <env_name>-env
+    data_dirs = [path for path in os.listdir(data_path) if os.path.join(data_path, path) and "_env" in path]
+    data_paths = []
+    # if there are dirs in data_path, each dir is assumed to hold data for a different env
+    if data_dirs:
+        data_paths.extend([os.path.join(data_path, dir_name) for dir_name in data_dirs])
+    else:
+        # if no dirs in data_path, this run in for single env
+        data_paths.append(data_path)
```

Comment:
I added this conditional in case someone wanted to run script on just data from one environment. I will update the readme with this possiblity

Reply:
Note that it is just another case to test and maintain, since you won't have the environment name.  My preference would be to leave it out and simplify unless it is needed, but this is just an opinion.

Reply:
yeah, I agree. Especially cause we only want this script to be mostly used by another automated script.